ci/build_alpine_container/README.md

```diff
@@ -0,0 +1,17 @@
+This directory contains a Dockerfile that builds an envoy image
```

Comment:
@mattklein123 Directly clicking on the Dockerfile link above is encoding the `~` char and hitting a 404 - please copy paste the link instead.. thanks!